tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: tilt_joint
    joint_speed: 0
    motor:
        id: 3
        init: 512
        min: 0
        max: 1023
